src/Caching/Memory/ref/Microsoft.Extensions.Caching.Memory.netcoreapp.cs

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.Extensions.Caching.Distributed
```

Comment:
The old ref assembly files stick around?

Reply:
ah yes, I need to clean those up.